charts/rancher-gke-operator/1.0.300/questions.yaml

```diff
@@ -0,0 +1 @@
+rancher_min_version: 2.5.8-patch1
```

Comment:
Should be 2.5.10-patch1, as Rancher codebase won't support this version until then. We don't want rancher v2.5.9 eventually getting this from release-v2.5 when 2.5.10 goes out.

Reply:
done